e2e/e2ek6.test.js

```diff
@@ -0,0 +1,307 @@
+                              {
+                                  "label": "event_time",
+                                  "name": "event_time",
+                                  "picklistValues": [],
+                                  "type": "DateTime"
+                              },
+                              {
+                                  "label": "event_time_microseconds",
+                                  "name": "event_time_microseconds",
+                                  "picklistValues": [],
+                                  "type": "DateTime64(6)"
+                              },
+                              {
+                                  "label": "query_start_time",
+                                  "name": "query_start_time",
+                                  "picklistValues": [],
+                                  "type": "DateTime"
+                              },
+                              {
+                                  "label": "query_start_time_microseconds",
+                                  "name": "query_start_time_microseconds",
+                                  "picklistValues": [],
+                                  "type": "DateTime64(6)"
+                              },
+                              {
+                                  "label": "initial_query_start_time",
+                                  "name": "initial_query_start_time",
+                                  "picklistValues": [],
+                                  "type": "DateTime"
+                              },
+                              {
+                                  "label": "initial_query_start_time_microseconds",
+                                  "name": "initial_query_start_time_microseconds",
+                                  "picklistValues": [],
+                                  "type": "DateTime64(6)"
+                              }
+                          ],
+                          "type": "datetime"
+                      }
+                  ],
+                  "limit": 100,
+                  "metrics": [],
+                  "mode": "list",
+                  "orderBy": [],
+                  "table": "query_log",
+                  "timeField": "event_time",
+                  "timeFieldType": "DateTime"
+              },
+              "queryType": "builder",
+              "rawSql": "SELECT memory_usage, event_time FROM system.\"query_log\" WHERE   ( event_time  >= $__fromTime AND event_time <= $__toTime ) LIMIT 100",
+              "refId": "A",
+              "meta": {
+                  "timezone": "America/Denver"
+              },
+              "datasourceId": 2922,
+              "intervalMs": 30000,
+              "maxDataPoints": 638
+          }
+      ],
+      "from": "1686147681613",
+      "to": "1686169281613"
+    };
+
+    const res = http.post('http://localhost:3000/api/ds/query/', JSON.stringify(queryData), {
```

Comment:
Same question as above for this

Reply:
I am going to match the [grafana `configurePanel` test](https://github.com/grafana/grafana/blob/d6c468c1c24fa70e2a0cfab6e5f8a98ed259e13d/packages/grafana-e2e/src/flows/configurePanel.ts#L112) and check the response of the POST request here.